test-npm/README.md

```diff
@@ -0,0 +1,10 @@
+# Test pkg.json against NPM's read-package-json library
```

Comment:
This `test-npm/` micro-project is just a way to get an idea of how close `pkg.json` is to `package.json`. Ideally it would be a strict subset but probably can't be. Any divergences will be reflected in the test files here.